packages/bsky/src/api/app/bsky/unspecced/getTimelineSkeleton.ts

```diff
@@ -14,27 +14,19 @@ export default function (server: Server, ctx: AppContext) {
       const db = ctx.db.db
       const { ref } = db.dynamic
 
-      const feedService = ctx.services.feed(ctx.db)
       const graphService = ctx.services.graph(ctx.db)
 
-      const followingIdsSubquery = db
-        .selectFrom('follow')
-        .select('follow.subjectDid')
-        .where('follow.creator', '=', viewer)
-
       const keyset = new FeedKeyset(
         ref('feed_item.sortAt'),
         ref('feed_item.cid'),
       )
       const sortFrom = keyset.unpack(cursor)?.primary
 
-      let feedItemsQb = feedService
-        .selectFeedItemQb()
-        .where((qb) =>
-          qb
-            .where('originatorDid', '=', viewer)
-            .orWhere('originatorDid', 'in', followingIdsSubquery),
-        )
+      const followBuilder = ctx.db.db
+        .selectFrom('follow')
+        .innerJoin('feed_item', 'feed_item.originatorDid', 'follow.subjectDid')
```

Comment:
Out of curiosity, do you know if switching the base table versus the join table makes any difference to postgres?  I think of this query as "selecting feed items" though of course the queries are equivalent 👍

Reply:
I don't believe it does i'll switch this back to being off of feed item 👍